decode_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"time"
 
 	. "gopkg.in/check.v1"
-	"gopkg.in/yaml.v3"
+	"github.com/setrofim/yaml"
```

Comment:
Do not change the package name.

Reply:
Ah, my apologies, that commit was not supposed to be added to the pull. Now reverted.